### PR TITLE
:ambulance: general hotfix

### PR DESCRIPTION
**Pull request by HUSKUT**
no ticket but i am very productive anyway!!!!!

### CODE REVIEW

**Comment by SophiaH67** [comment]
I'm guessing this is not intended?

Screenshot_20230125-120212_Expo Go

File: components/Loading.tsx [hunk 1]
```diff
@@ -1,12 +1,40 @@
...
+
+    await Logout(navigation.navigate);
+  }
+
+  useEffect(() => {
+    getLoggedInUser().then(setUser).catch(alert);
+    // .catch(alert); - Removed because this fires for normal users as well
+    // and they don't have access to the companies endpoint, so it would
+    // always show a confusing error message.
+  }, [refresh]);
```
Comment (SilGosker) [inline components/Loading.tsx]:
```suggestion
    const [refresh, setRefresh] = useState(Math.random());

    async function CustomLogout() {
        setRefresh(Math.random);
        await Logout(navigation.navigate);
    }

    useEffect(() => {
    }, [refresh]);
```

File: components/Loading.tsx [hunk 2]
```diff
@@ -1,12 +1,35 @@
...
+  const [user, setUser] = useState<User>();
+  const [refresh, setRefresh] = useState(Math.random());
+
+  async function CustomLogout() {
+    setRefresh(Math.random());
+
+    await Logout(navigation.navigate);
+  }
+
+  useEffect(() => {}, [refresh]);
```
Comment (SophiaH67) [inline components/Loading.tsx]: What's the use of this useEffect? To me it looks like there is nothing happening in the effect(empty function)

**Review by SophiaH67 — APPROVED** [review]

File: components/Loading.tsx (hunk 2 above)
Comment (HUSKUT) [reply components/Loading.tsx]: is funny

**Review by SophiaH67 — APPROVED** [review]

**Pull request merged**